pyro/infer/inspect.py

```diff
@@ -339,6 +366,16 @@ def generate_graph_specification(model_relations: dict) -> dict:
         rv for rv in model_relations["sample_sample"] if rv not in plate_rvs
     ]  # RVs which are in no plate
 
+    params = set()
```

Comment:
nit: let's give this a consistent type, either set or list. we're moving towards checking with mypy and we want to make sure this code won't cause later type errors

Reply:
Yes, I resolved it by keeping set only.